source/community/release_notes/palm.rst

```diff
@@ -23,14 +63,109 @@ Learner Experiences
 Instructor Experiences
 **********************
 
+New Visual Problem Editor
+=========================
+
+The release includes an improved problem-authoring experience with an interactive editor. Writing markdown 
+code is no longer necessary. But, the advanced mode is still available, maintaining the ability to write and edit 
+OLX XML.
+
+The Open edX wiki page `[2U] New Visual Problem Editor <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2023/04/07/3724312593/2U+New+Visual+Problem+Editor>`_ 
+provides a brief explanation of what has changed in the problem editor. Updated detailed instructions on writing 
+problems can be found in section `8.4. Working with Problem Components <https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/course_components/create_problem.html#working-with-problem-components>`_ of the Building and Running an edX Course documentation.
+
+The visual problem editor is an update to the Course Authoring Micro-frontend first released in Olive and is enabled by 
+default in Palm. To enable the Visual Problem Editor, add the waffle flag 
+:code:`new_core_editors.use_new_problem_editor` and set the value to “Yes” for all users.
+
+New ORA Grading Experience
+==========================
+
+In this new on-platform grading experience one can easily preview common file types, assign rubric values, provide 
+comments, and coordinate grading with all members of the course teams. Complete documentation is in section 
+`10.26.4. Staff Grading for Open Response Assignments 
+<https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/open_response_assessments/ORA_Staff_Grading.html#staff-grading-for-open-response-assignments>`_  of the Building and Running an edX Course documentation. 
+
+The new ORA grading experience depends on the ORA Grading Micro-frontend, which was included as an experimental 
+feature in Olive. To turn on the feature, add the feature flag :code:`openresponseassessment.enhanced_staff_grader`.
+
+New Bulk Email Experience
+=========================
+
+The new bulk email experience for instructors is enabled automatically in Palm. It allows instructors to schedule bulk 
+emails, instead of sending them immediately. 
```

Comment:
Is there more information on this? Screenshots maybe? How could I test it?

Reply:
@pdpinch I put in some info about the bulk email experience in the "smol" blog post: https://docs.google.com/document/d/1IUIlsJX4EtmIBuJdbcBj-LQsGHn8wQyLkByiYj5Sips/edit#

Reply:
I added a screenshot and also tried to explain the situation with scheduled sending, which is not easy to set up.